### DISCUSSION TITLE
Dashboard Panel Sizing/Resizing

### DISCUSSION THREAD

**Discussion by DaveGerrish** [grafana/grafana#39242] [Dashboards, Panels & Visualizations]
It would be great to be able to size the dashboard panels to set sizes - without having to guess when comparing to another panel or by having to query the size of another panel (via JSON inspection) and then edit the JSON of the new panel to make them the same.
A simple hover tip when dragging showing width and height wold be good.
Also a grid you could turn off/on. Also snap to grid customisation.
Anything to make sizing panels easier/simpler.
 
[apologies if this is not the right format for posts - just copied my ER request I sent to Broadcom as am using their OI dashboards and have this issue]

**Comment by C0casio45** [comment]
any update on this post ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
